charts/posthog/templates/events-deployment.yaml

```diff
@@ -118,7 +118,7 @@ spec:
         livenessProbe:
           failureThreshold: {{ .Values.web.livenessProbe.failureThreshold }}
           httpGet:
-            path: /_health/
+            path: /_livez/
```

Comment:
@guidoiaquinti would you mind helping me with the release process here?

This will actually not fail on old posthog images, as it redirects to login page (rather than returning a 404), but `/_health/` was testing more.

Not so relevant for liveness as I don't think it should be testing much more than returning a response, but for readiness is will be an issue.

How should this be managed?

Reply:
Good question. I don't think there's a clear winner here, I trust your judgment. Personally I think we should probably cut a new PH release, reference it here as default and then merge this PR.